<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>患者信息界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewButton">
        <property name="text">
         <string>视图</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>

    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>

      <widget class="QWidget" name="leftWidget">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QPushButton" name="personalInfoButton">
          <property name="styleSheet">
           <string>background-color: #333333; color: white; font-size: 16px; border: 1px solid #555555;</string>
          </property>
          <property name="text">
           <string>个人信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="medicalRecordsButton">
          <property name="styleSheet">
           <string>background-color: #333333; color: white; font-size: 16px; border: 1px solid #555555;</string>
          </property>
          <property name="text">
           <string>医疗记录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="surgeryProgressButton">
          <property name="styleSheet">
           <string>background-color: #333333; color: white; font-size: 16px; border: 1px solid #555555;</string>
          </property>
          <property name="text">
           <string>手术进度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="feedbackButton">
          <property name="styleSheet">
           <string>background-color: #333333; color: white; font-size: 16px; border: 1px solid #555555;</string>
          </property>
          <property name="text">
           <string>消息和反馈</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="knowledgeBaseButton">
          <property name="styleSheet">
           <string>background-color: #333333; color: white; font-size: 16px; border: 1px solid #555555;</string>
          </property>
          <property name="text">
           <string>知识库</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>消息通知</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="tableWidgetContainer">
       <layout class="QVBoxLayout" name="tableLayout">
        <!-- 搜索框和按钮 -->
        <item>
         <layout class="QHBoxLayout" name="searchLayout">
          <item>
           <widget class="QLineEdit" name="searchBox">
            <property name="placeholderText">
             <string>搜索框</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>

        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="columnCount">
           <number>4</number>
          </property>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>拍摄时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>备注</string>
           </property>
          </column>
         </widget>
        </item>

       <!-- 分页布局 -->
        <item>
         <layout class="QHBoxLayout" name="pageControlsLayout">
          <item>
           <spacer name="leftSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="firstPageButton">
            <property name="text">
             <string>|&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="previousPageButton">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="pageSpacerLeft">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="pageLabel">
            <property name="text">
             <string>1/1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="pageSpacerRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="nextPageButton">
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="lastPageButton">
            <property name="text">
             <string>&gt;|</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="rightSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </spacer>
          </item>
         </layout>
        </item>

       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>咨询</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
